vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/DisplayTab.java

```diff
@@ -158,6 +163,35 @@ private static void verifyBoutiquesTool(BoutiquesApplication boutiquesTool)
         if (boutiquesTool.getAuthor() == null) {
             throw new ApplicationImporterException("Boutiques file must have an author");
         }
+         checkvipdot(boutiquesTool);
+    }
+    
+    /**
+     * display warning message if any.
+     *
+     * @param application BoutiquesApplication object to cehck warning message 
+     * **/
+    private static void checkvipdot(BoutiquesApplication application) {
+        Set<String> commandLineFlags = application.getCommandLineFlag();
+        Set<String> vipDotInputIds = application.getVipDotInputIds();
+        Set<String> commonValues = new HashSet<>(application.getVipDotInputIds());
+        
+        commonValues.retainAll(commandLineFlags);
+
+        if (!commonValues.isEmpty()) {
+            String warningMessage = "<b>" + String.join(", ", commonValues) + "</b> appears as command-line flag input(s), it should not be included in Dot iteration. Importing it may cause functionality issues, although the application will still be imported.";
+            Layout.getInstance().setWarningMessage(warningMessage);
+        }
+
+        // Extract the IDs from inputs
+        Set<String> inputIds = application.getInputs().stream().map(BoutiquesInput::getId).collect(Collectors.toSet());
```

Comment:
You may add this as a BoutiquesApplication method. (optionally)

Reply:
Added the getinputIds method to BoutiquesApplication